tests/test_sshfsmount.cpp

```diff
@@ -88,7 +156,7 @@ struct SshfsMount : public mp::test::SftpServerTest
 TEST_F(SshfsMount, throws_when_sshfs_does_not_exist)
 {
     bool invoked{false};
-    auto request_exec = make_exec_that_fails_for("which sshfs", invoked);
+    auto request_exec = make_exec_that_fails_for({"which sshfs"}, invoked);
```

Comment:
Undo that change, please - and all the similar ones below.

Reply:
Done.